aiidalab_widgets_base/viewers.py

```diff
@@ -34,32 +34,22 @@ def registration_decorator(widget):
     return registration_decorator
 
 
-def viewer(obj, downloadable=True, **kwargs):
```

Comment:
We can't be passing this attribute to the viewers in general because most of them do not have it, and then pass it up to the ipywidget parent which results in warnings. Since this parameter is only used in `DictViewer` and `FolderDataViewer` I think it is okay to get rid of it, but please correct me if I am wrong.

Reply:
The problem maybe the viewer return from calling `viewer` directly will not be exactly the same as `DictViewer`. Where the `downloadable` can not pass to the viewer anymore. 
I think it is fine to get rid of it but for the `DictViewer` the `downloadable` need to get from `kwargs`.

Reply:
Sorry, I am not sure I got your comment. Do you want me to change anything?

> The problem maybe the viewer return from calling viewer directly will not be exactly the same as DictViewer. Where the downloadable can not pass to the viewer anymore.

Yes. But other viewers may have other input parameters and we don't support them either, so I don't see why `Downloadable` should be privileged in any way.

Reply:
> Sorry, I am not sure I got your comment. Do you want me to change anything?

Never mind, I misunderstood the usage of the viewer in the first place. I was asking for changing `__init__` of `DictViewer`, instead of passing `downloadable` but passing `downloadable` through `kwargs`. 

It is not necessary, and your change is all good. The expected way of using the `viewer` function in the AiiDAlab is by `AiidaNodeViewecWidget`, which only passes the `orm.Node` without any parameters.